.github/workflows/deploy-gcp-tests.yml

```diff
@@ -234,7 +239,15 @@ jobs:
           --zone ${{ env.ZONE }}
           sleep 60
 
+      # SSH into the just created VM, and create a Docker container to run the incoming test 
+      # from ${{ inputs.test_id }}, then create a docker volume with the recently attached disk.
+      # The disk will be mounted in ${{ inputs.root_state_path }}/${{ inputs.zebra_state_dir }}.
+      #
+      # This step mounts the volume only when a single cached state is needed, in this case
+      # the cached state from Zebra.
       - name: Run ${{ inputs.test_id }} test
+        # Validate we only need zebra's cache and not lightwalletd's cache
+        # if: ${{ inputs.zebra_state_dir }} != '' && ${{ inputs.lwd_state_dir }} == ''
```

Comment:
Question: I'm not sure what this comment means?

Reply:
Fixed by: [fix(ci): remove not used variable lwd_state_dir](https://github.com/ZcashFoundation/zebra/pull/4268/commits/b19888cde86dc5e1632b7ea5757538e2075a9611)